syncd/SaiSwitchInterface.h

```diff
@@ -91,8 +91,8 @@ namespace syncd
             virtual std::set<sai_object_id_t> getWarmBootNewDiscoveredVids();
 
             virtual void onPostPortCreate(
-                    _In_ sai_object_id_t port_rid,
-                    _In_ sai_object_id_t port_vid) = 0;
+                    _In_ sai_object_id_t* port_rid,
+                    _Out_ size_t count) = 0;
 
```

Comment:
function naming dont correspond to parameters

Reply:
Done